src/index.js

```diff
@@ -18,9 +18,20 @@ import 'babel-polyfill';
 import fs from 'fs';
 import path from 'path';
 import vorpal from 'vorpal';
-import { version } from '../package.json';
+import semver from 'semver';
+import { version, engines } from '../package.json';
 import config from './utils/config';
 
+const nodeVersionError = (required, actual) =>
```

Comment:
Prefer imperative format for functions, e.g. `formatNodeVersionError` or `constructNodeVersionErrorMessage`.